src/main/java/com/eternalcode/combat/config/implementation/PluginConfig.java

```diff
@@ -50,6 +51,18 @@ public static class Settings {
 
         @Description({ " ", "# From which level should place blocks be blocked?" })
         public int blockPlaceLevel = 40;
+
+        @Description({ " ", "# Should the option below be enabled?" })
+        public boolean enableDamageCauses = true;
+
+        @Description({ " ", "# After what type of damage the player should get a combat log?",
+        "# You can find a list of all stocks here: https://hub.spigotmc.org/javadocs/spigot/org/bukkit/event/entity/EntityDamageEvent.DamageCause.html" })
+        public List<EntityDamageEvent.DamageCause> damageCauses = new ImmutableList.Builder<EntityDamageEvent.DamageCause>()
```

Comment:
Domyślnie daj lawa, mob, fire i ewentualnie obrażenia od np. kaktusa itd, oraz opisz w komentarzu że jak ktoś nie chce czegoś to żeby usunął z listy po prostu.